samples/matter/light_switch/README.rst

```diff
@@ -344,7 +344,7 @@ After building this and the :ref:`Matter Light Bulb <matter_light_bulb_sample>`
    During the commissioning process, write down the values for the light switch node ID and the light bulb node ID (or IDs, if you are using more than one light bulb).
    These IDs are going to be used in the next steps (*<light_switch_node_ID>* and *<light_bulb_node_ID>*, respectively).
 #. Use the :doc:`CHIP Tool <matter:chip_tool_guide>` ("Writing ACL to the ``accesscontrol`` cluster" section) to add proper ACL for the light bulb device.
-   Run the following command, with *<light_switch_node_ID>* and *<light_bulb_node_ID>* values from the previous step about commissioning:
+   If you are using only one light bulb device, run the following command, with *<light_switch_node_ID>* and *<light_bulb_node_ID>* values from the previous step about commissioning:
```

Comment:
What if I'm running more? Line 24 mentions "at least one", which means that we also need a case here about multiple light bulb devices. What about "For each light bulb device you are using"?

Reply:
Not exactly, the listed command (`chip-tool accesscontrol write acl...`) is valid only in case we have exactly one light bulb (unicast communication). When more than one light bulb is engaged (multicast), this step is supposed to be skipped and the following paragraph should be considered (`If you are using more than one light bulb device`).

Reply:
OK, so we need to rewrite this whole part, unfortunately.

Reply:
```

#. Use the :doc:`CHIP Tool <matter:chip_tool_guide>` ("Writing ACL to the ``accesscontrol`` cluster" section) to add proper ACL for the light bulb device.
   Depending on the number of the light bulb devices you are using, use one of the following commands, with *<light_switch_node_ID>* and *<light_bulb_node_ID>* values from the previous step about commissioning:

   * If you are using only one light bulb device, run the following command for the light bulb device:

   .. parsed-literal::
      :class: highlight

      chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"fabricIndex": 1, "privilege": 3, "authMode": 2, "subjects": [<light_switch_node_ID>], "targets": [{"cluster": 6, "endpoint": 1, "deviceType": null}, {"cluster": 8, "endpoint": 1, "deviceType": null}]}]' <light_bulb_node_ID> 0

   * If you are using more than one light bulb device, connect all devices to the multicast group by running the following command for each device, including the light switch:
   
   .. parsed-literal::
      :class: highlight

      chip-tool tests TestGroupDemoConfig --nodeId <node_ID>
   
     Use the *<node_ID>* values from the commissioning step.
```